libs/SDL2-2.0.12/include/SDL_config_psp.h

```diff
@@ -123,10 +123,10 @@
 /* Enable the stub audio driver (src/audio/psp/\*.c) */
 #define SDL_AUDIO_DRIVER_PSP    1
 
-/* PSP video dirver */
```

Comment:
The folder libs contains dependencies that are copied from the source. Please do not change these files, as it will make upgrading dependencies harder.